### ISSUE TITLE
Insert tables with padding and border visible on text

### ISSUE THREAD

**Issue by olairmarcos** [iionly/event_calendar#4]
Insert tables with padding and border visible on text

Hello, I added the plugins (Table) and configured so that you can create tables in the text through the CKeditor_extend editor.
When I'm creating text with tables, I see the table and I set the style by setting the table properties box. All normal, the table has the edge and spacing of the cell perfectly. See image below

![2016-05-07_110247](https://cloud.githubusercontent.com/assets/8938718/15092964/8af56952-144f-11e6-8ea3-a23f74c27d90.jpg)

![2016-05-07_110619](https://cloud.githubusercontent.com/assets/8938718/15092966/90689b98-144f-11e6-804c-0d9a0abf472e.jpg)

When I saved the text, I see that the border style has been deleted and the spacing of the cell (cellpadding) was also deleted. See image below.

![2016-05-07_111103](https://cloud.githubusercontent.com/assets/8938718/15092971/9b44dc98-144f-11e6-8175-ade62f433ba3.jpg)

I need the table border and spacing (cellpadding) is visible after the description page is saved in event_calendar.
I tried to look for where you can be wrong, but I can not find the solution.
This problem only appears in event_calendar plugin.
Can you help me?
Thanks for listening.

**Comment by iionly** [comment]
Are you really sure that it happens with the long description field of the Event Calendar plugin? I would guess it rather happens with all longtext input fields because the Htmlawed plugin will very likely filter out the table style tags from the input. Then I can't do anything within the Event Calendar plugin to prevent that. Unfortunately, I also can't tell you how to change the config of the allowed tags for the Htmlawed plugin (Elgg has a plugin hook to allow for altering the config). Maybe someone at the Elgg community site can help you with setting up Htmlawed correctly to allow for keeping the table styles

**Comment by olairmarcos** [comment]
Hello, I tried to momentarily disable Htmlawed to test. It did not solve the problem. In Blog and Pages plugin, the table is working perfectly. Thanks for answering.

**Comment by iionly** [comment]
Try the following:

In the file mod/event_calendar/views/default/object/event_calendar.php change

``` php
    if ($event->long_description) {
        $body .= '<div class="mtm">' . $event->long_description . '</div>';
    } else if ($event->description) {
```

into

``` php
    if ($event->long_description) {
        $body .= elgg_view('output/longtext', array('value' => $event->long_description, 'class' => 'mtm'));
    } else if ($event->description) {
```

Does this help?

**Comment by olairmarcos** [comment]
Yes, now is working properly, the border and style (cellpadding and cellspacing) appear in the text. see figure below.

![2016-05-07_161405](https://cloud.githubusercontent.com/assets/8938718/15094048/4dd7e96c-146f-11e6-8173-19007c29ed5a.jpg)

Thanks a lot for the help. God is always generous with you. Good weekend.

**Comment by iionly** [comment]
Fixed in version 2.3.1.
